core/node/da_clients/Cargo.toml

```diff
@@ -43,6 +44,7 @@ backon.workspace = true
 url.workspace = true
 
 # Celestia dependencies
+eq-sdk = { path = "../../../../eq-service/sdk", package = "eq-sdk" }
```

Comment:
Will publish on crates shortly: https://github.com/celestiaorg/eq-service/sdk

Reply:
https://crates.io/crates/eq-sdk is now up, although we likely need a patch release before this PR is merge-able

Reply:
0.1.1 is up, and the required re-exported `eq-common` is too
https://crates.io/crates/eq-common/versions